### ISSUE TITLE
검색 오류

### ISSUE THREAD

**Issue by dve1** [gdl-blue/imitated-seed-2#137]
검색하는데 너무 오래 걸립니다.

**Comment by dve1** [comment]
searchd_fail 오류가 나네요

**Comment by wikiengine** [comment]
search.js 실행하셨나요?

**Comment by wikiengine** [comment]
안하셨으면 config.json에
`,“search_autostart”:“true”`
추가하시거나 node search추가로 실행하세요

**Comment by dve1** [comment]
```
node:events:492
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRNOTAVAIL: address not available 127.5.5.5:25005
    at Server.setupListenHandle [as _listen2] (node:net:1855:21)
    at listenInCluster (node:net:1920:12)
    at doListen (node:net:2069:7)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
Emitted 'error' event on Server instance at:
    at emitErrorNT (node:net:1899:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
  code: 'EADDRNOTAVAIL',
  errno: -49,
  syscall: 'listen',
  address: '127.5.5.5',
  port: 25005
}

Node.js v20.10.0
```
오류가 납니다.

**Comment by wikiengine** [comment]
> ```
> node:events:492
>       throw er; // Unhandled 'error' event
>       ^
> 
> Error: listen EADDRNOTAVAIL: address not available 127.5.5.5:25005
>     at Server.setupListenHandle [as _listen2] (node:net:1855:21)
>     at listenInCluster (node:net:1920:12)
>     at doListen (node:net:2069:7)
>     at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
> Emitted 'error' event on Server instance at:
>     at emitErrorNT (node:net:1899:8)
>     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
>   code: 'EADDRNOTAVAIL',
>   errno: -49,
>   syscall: 'listen',
>   address: '127.5.5.5',
>   port: 25005
> }
> 
> Node.js v20.10.0
> ```
> 
> 오류가 납니다.

추가하시거**나**
